modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/editor/ClassicEditor.js

```diff
@@ -78,6 +84,11 @@ const ClassicEditor = forwardRef(
 							},
 							noSnapshot: true,
 						});
+						const editable = editor.editable();
+						editable.editor.document.$.activeElement.setAttribute(
```

Comment:
`activeElement` is a bit scary here. At the time this code resolves, it's probably not possible to know for sure what element will have focus. Any other code on page or even user interaction can shift focus. Is the editable area intended target for `aria-required`? If yes, can we use `editable.setAttribute()` like [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/editor/BalloonEditor.js#L50-L54)?

Reply:
I guess now I understand your point, @markocikos. I apologize for my delay in doing it.
I changed the fix, and now I am not using the editable, but looking for the cke body because that is where I want to put the aria-required.